keyboards/binepad/candypad/rules.mk

```diff
@@ -0,0 +1,9 @@
+# Copyright 2023 binepad (@binepad)
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+CUSTOM_MATRIX = yes
```

Comment:
this should be `lite` and the custom matrix should be re-implemented to use the lite implementation.  This is much simpler, handles debounce, and is the preferred implementation method, anymore. 

https://docs.qmk.fm/custom_matrix#lite